config/schema.error.yaml

```
@@ -0,0 +1,575 @@
+    stage: general
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::GLOBAL_CONFIG_UNABLE_TO_READ:
+    title: 'Unable to read the `.magento.env.yaml` file'
+    suggestion: 'Unable to read the `./.magento.env.yaml` file. Check file permissions.'
+    stage: general
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::GLOBAL_CONFIG_UNABLE_TO_READ_SCHEMA_YAML:
+    title: 'Unable to read the `.schema.yaml` file'
+    suggestion: ''
+    stage: general
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::WARN_CONFIG_PHP_NOT_EXISTS:
+    title: 'File app/etc/config.php does not exist'
+    stage: build
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_UNSUPPORTED_BUILDS_OPTION_INI:
+    title: 'The ./build_options.ini file is no longer supported'
+    stage: build
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_MISSED_MODULE_SECTION:
+    title: 'The modules section is missing from the shared config file'
+    stage: build
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_CONFIGURATION_VERSION_MISMATCH:
+    title: 'Configuration is not compatible with this version of Magento'
+    stage: build
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_SCD_OPTIONS_IGNORANCE:
+    title: 'Scd options ignored'
+    stage: build
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_CONFIGURATION_STATE_NOT_IDEAL:
+    title: 'The configured state is not ideal'
+    stage: build
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_BALER_CAN_NOT_BE_USED:
+    title: 'Baler JS bundling cannot be used'
+    stage: build
+    step: run-baler
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_REDIS_SERVICE_NOT_AVAILABLE:
+    title: 'Cache is configured for a Redis service that is not available. Configuration will be ignored.'
+    stage: deploy
+    step: 'pre-deploy:cache'
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_WRONG_SPLIT_DB_CONFIG:
+    title: 'The configured state is not ideal'
+    stage: deploy
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_DIR_NESTING_LEVEL_NOT_CONFIGURED:
+    title: 'The directory nesting level value for error reporting has not been configured'
+    stage: deploy
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_NOT_CORRECT_LOCAL_XML_FILE:
+    title: 'Invalid file config ./pub/errors/local.xml'
+    stage: deploy
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_ADMIN_DATA_IGNORED:
+    title: 'Admin data is used to create an admin user during initial installation only and is ignored during upgrade process'
```

```suggestion
    title: 'Admin data is used to create an admin user during initial installation only. Any changes to Admin data are ignored during the upgrade process.'
```

 upgrades and redeployments?  -- How do you change Admin data after installation?